src/BotPlutusInterface/Config.hs

```diff
@@ -285,7 +286,7 @@ loadPABConfig fn = do
   case confE of
     Left err -> return $ Left $ "PABConfig: " <> fn <> ": " <> err
     Right conf@PABConfig {pcProtocolParamsFile, pcNetwork, pcCliLocation} -> do
-      pparamsE <- readProtocolParametersJSON (toString pcProtocolParamsFile)
+      pparamsE <- readProtocolParametersJSON (toString pcProtocolParamsFile) `catch` (\(e :: IOException) -> pure $ Left (show e))
```

Comment:
Could we include the `catch` in the `readProtocolParametersJSON`? It already returns an `Either`, so it would make sense, if IO errors would also be caught there.

Reply:
This is the place:
https://github.com/mlabs-haskell/bot-plutus-interface/blob/master/src/PlutusConfig/Cardano/Api/Shelley.hs#L279